Create 2020-04-15-all-hands.md (#267)
hyphacoop/organizing: _posts/meeting-notes/2020-04-15-all-hands.md

### Task Board Review

- Archive DONE tasks :tada:
	- #240 done
- Review [HIGH `[priority-★★★]`][l-pri-hi] tasks :fire:
	- #95 finished today IP agreement will be applied to anyone that we hire
	    - need respond from dc
	- #201 needs review from uv
	- #35 going to wait a couple weeks for things to setle down for a bit
	- #244 keep an eye on it to make sure it is renewed next week
	- #133 still have to do this and hire non-members
	    - don't have IP agreement for non-member yet
	- #77 pull request opened in the handbook, announcment on twitter
	- #261 we might get the 75% support applied for the 10% subsidy program
	    - updated requirement in the thread
	    - bl: is it 75% on the payroll or on Jan and Feb?
	        - need research?
	- #262 shortlinks api broke, el needs to fix
	    - yj: may scrap rebrandly completely
	- #102 no updates
	- #250 did some hiring things today next step create hiring process
	    - bl: doing research on domestic and remote and need to confirm with accuntent
- Triage NEW tasks :new:
	- None to triage
- Assign [UNASSIGNED][l-none] tasks :briefcase:
	- ...

> - Any isues we should discuss?

Reviewer: ```suggestion
- Any issues we should discuss?
```

>     - #131 talked about yesterday and deplicate some internal addresses

Reviewer: ```suggestion
    - #131 talked about yesterday and duplicate some internal addresses
```

        - document email setup
        - update inventory
        - yj to run show and tell on email
        - using el's infra as email provider
        - all tasks in subtasks in the issue
    - #6 on website

>     - #5 data retention on next website version

Reviewer: What are these numbers @ASoTNetworks ?

Author: website

    - dc: StART hiring
    - dc: COVID also lead to needing to resolve OHSA health and safety requirements
    - bl: need to hire prioritized other tasks

### Project Updates

- Opportunity: Mozilla MVP Lab
    - bl: lluna joined their Slack with 2000 people 😱, pushed their deadline to respond
- StART 
    - dc: sent email, next steps dc as main person while pc is away, ideally 2 weeks to get someone up to speed
- Qm 
    - uv: no much changes want to know data retention, legal, ... for this Friday
- Virtual Grounds
    - dc: just needs to get paid, will follow up. workshop went well!
    - dc: ppl were so interested in dweb stuff
    - dc: ppl exploring new emerging tech... others in the org may be better fit in future engagements
- Distributed Press
    - uv: project meets once a week slow and stedy progress
    - uv: MVP a grant to apply to when avaliable from web monetization program
    - bl: https://github.com/benhylau/distributed.press/blob/master/interviews.md
    - bl: https://github.com/benhylau/distributed.press/blob/master/values.md
    - bl: how DP can address the issues 
    - bl: aligned with Hypha values
- Aether
    - bl: finally working on this again

>     - bl: burak refocusing on COVID, build a chat async comms and thinking on market position

Reviewer: ```suggestion
    - bl: burak refocusing on Aether after supporting COVID-19 efforts, building a chat with sync comms and thinking on market position
```
